<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1026</width>
    <height>699</height>
   </rect>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::DefaultContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string>Labeling Tool</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="imageTab">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>40</y>
      <width>710</width>
      <height>540</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>2</number>
    </property>
    <property name="usesScrollButtons">
     <bool>false</bool>
    </property>
    <property name="tabBarAutoHide">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="imageRawTab">
     <attribute name="title">
      <string>Raw</string>
     </attribute>
     <widget class="QLabel" name="imageLabelTabRaw">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>706</width>
        <height>509</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="autoFillBackground">
       <bool>true</bool>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="scaledContents">
       <bool>false</bool>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="imageSegTab">
     <attribute name="title">
      <string>Seg</string>
     </attribute>
     <widget class="QLabel" name="imageLabelTabSeg">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>706</width>
        <height>509</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="autoFillBackground">
       <bool>true</bool>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="scaledContents">
       <bool>false</bool>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="imageBlobTab">
     <attribute name="title">
      <string>blob</string>
     </attribute>
     <widget class="QLabel" name="imageLabelTabBlob">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>706</width>
        <height>509</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="autoFillBackground">
       <bool>true</bool>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="scaledContents">
       <bool>false</bool>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QGroupBox" name="rightUpperGroupBox">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>0</y>
      <width>261</width>
      <height>331</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>321</height>
     </size>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::NoContextMenu</enum>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="title">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
    <widget class="QListWidget" name="labelList">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>180</y>
       <width>221</width>
       <height>121</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="labelListTitle">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>160</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Label List:</string>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="addLabelButton">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>90</y>
       <width>151</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Add Label</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="keyBindEdit">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>60</y>
       <width>121</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="keyBindTitle">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>60</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Keybind:</string>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLineEdit" name="labelNameEdit">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>30</y>
       <width>121</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="labelNameTitle">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Label Name:</string>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="removeLabelButton">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>120</y>
       <width>151</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Remove Label</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="upperGroupBox">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>10</y>
      <width>710</width>
      <height>30</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QLabel" name="dbStatus">
     <property name="geometry">
      <rect>
       <x>480</x>
       <y>0</y>
       <width>221</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>dbStatus</string>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="imageIDCount">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>0</y>
       <width>341</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>image_id</string>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="rightBelowGroupBox">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>330</y>
      <width>261</width>
      <height>255</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QLabel" name="curImageLabelTitle">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>171</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Current Image Labels:</string>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QListWidget" name="curImageLabelsList">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>221</width>
       <height>121</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="prevImageButton">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>214</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Prev</string>
     </property>
    </widget>
    <widget class="QPushButton" name="nextImageButton">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>214</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Next</string>
     </property>
    </widget>
    <widget class="QPushButton" name="focusModeButton">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>170</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Focus Mode</string>
     </property>
    </widget>
    <widget class="QLabel" name="focusStatus">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>170</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>OFF</string>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="bottomGroupBox">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>590</y>
      <width>1021</width>
      <height>51</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QProgressBar" name="bottomProgressBar">
     <property name="geometry">
      <rect>
       <x>13</x>
       <y>13</y>
       <width>95</width>
       <height>25</height>
      </rect>
     </property>
     <property name="value">
      <number>24</number>
     </property>
    </widget>
    <widget class="QLabel" name="progressBarText">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>13</y>
       <width>311</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>progressbartext</string>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1026</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen_Workspace"/>
   </widget>
   <widget class="QMenu" name="menuDatabase">
    <property name="title">
     <string>Database</string>
    </property>
    <addaction name="actionCreate_New_Database"/>
    <addaction name="actionUpdate_Database"/>
   </widget>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>Tools</string>
    </property>
    <widget class="QMenu" name="menuBlob_Detector">
     <property name="title">
      <string>Blob Detector</string>
     </property>
     <addaction name="actionBlob_Detector_With_Face_Parsing"/>
    </widget>
    <widget class="QMenu" name="menuFace_Parsing_Tool">
     <property name="title">
      <string>Face Parsing Tool</string>
     </property>
     <addaction name="actionParse_Current_Image"/>
    </widget>
    <addaction name="menuFace_Parsing_Tool"/>
    <addaction name="menuBlob_Detector"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuDatabase"/>
   <addaction name="menuTools"/>
  </widget>
  <widget class="QStatusBar" name="statusbar">
   <property name="sizeGripEnabled">
    <bool>false</bool>
   </property>
  </widget>
  <action name="actionOpen_Workspace">
   <property name="text">
    <string>Open Workspace</string>
   </property>
  </action>
  <action name="actionCreate_New_Database">
   <property name="text">
    <string>Create New Database</string>
   </property>
  </action>
  <action name="actionBlob_Detector_With_Face_Parsing">
   <property name="text">
    <string>Blob Detector With Face Parsing</string>
   </property>
  </action>
  <action name="actionParse_Current_Image">
   <property name="text">
    <string>Parse Current Image</string>
   </property>
  </action>
  <action name="actionUpdate_Database">
   <property name="text">
    <string>Update Database</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
